py/sentry_streams/pipeline.py

```diff
@@ -87,3 +88,13 @@ class KafkaSink(WithInput):
 
     logical_topic: str
     step_type: StepType = StepType.SINK
+
+
+@dataclass
+class Map(WithInput):
+    """
+    A simple 1:1 Map, taking a single input to single output
+    """
+
+    function: str
```

Comment:
Did we ever  consider how to provide the product access to the context of the pipeline? If yes is it going to be part of a follow up PR? If yes please add a todo.

Reply:
Left TODO